impl/src/bitfield.rs

```diff
@@ -271,22 +325,96 @@ impl BitfieldStruct {
     /// amount of bytes to compactly store the information of all its bit fields.
     fn generate_struct(&self) -> TokenStream2 {
         let span = self.item_struct.span();
-        let attrs = &self.item_struct.attrs;
+        let attrs = self.get_attrs_no_debug();
         let vis = &self.item_struct.vis;
         let ident = &self.item_struct.ident;
         let size = self.generate_bitfield_size();
         let next_divisible_by_8 = Self::next_divisible_by_8(&size);
         quote_spanned!(span=>
             #( #attrs )*
             #[repr(transparent)]
-            #[allow(clippy::identity_op)]
+            #[allow(clippy::identity_op, unused_attributes)]
             #vis struct #ident
             {
                 bytes: [::core::primitive::u8; #next_divisible_by_8 / 8usize],
             }
         )
     }
 
+    /// Get whether or not `derive(Debug)` is included in the attributes
+    fn should_derive_debug(&self) -> bool {
```

Comment:
We should also provide a conditional `Debug` impl for cases like
```rust
#[bitfield]
#[cfg_attr(test, derive(Debug))]
pub struct MyBitfield { ... }
```
In this case we'd like to also only implement `Debug` if the given conditional (in this case `test`) is true.

Reply:
While implementing this in my own I found out that the Rust compiler already handles this for us. So nothing needs to be done for `cfg_attr` anywhere fortunately. :)
See also: https://github.com/Robbepop/modular-bitfield/pull/38